third_party/terraform/website/docs/guides/version_3_upgrade.html.markdown

```diff
@@ -53,19 +53,30 @@ so Terraform knows to manage them.
 
```

Comment:
I don't think I added every change in here, but I plan to go through all the docs and update them.